libraries/AP_Logger/LogStructure.h

```diff
@@ -1930,6 +1944,8 @@ struct PACKED log_PSC {
       "BAT", "QBfffffcf", "TimeUS,Instance,Volt,VoltR,Curr,CurrTot,EnrgTot,Temp,Res", "s#vvAiJOw", "F-000!/?0" },  \
     { LOG_CURRENT_CELLS_MSG, sizeof(log_Current_Cells), \
       "BCL", "QBfHHHHHHHHHHHH", "TimeUS,Instance,Volt,V1,V2,V3,V4,V5,V6,V7,V8,V9,V10,V11,V12", "s#vvvvvvvvvvvvv", "F-0CCCCCCCCCCCC" }, \
+    { LOG_CURRENT_CELLS2_MSG, sizeof(log_Current_Cells2), \
+      "BCL2", "QBfHH", "TimeUS,Instance,V13,V14", "s#vv", "F-CC" }, \
```

Comment:
num fields don't match, why didn't CI find this?

Reply:
It did.  The tests were clearly not passing on this PR.